### PR TITLE
updated readme to reflect current local deployment standards.

### PR DESCRIPTION
**Pull request by MM-USAF**
THis is an update to the readme file for new users to spin up a local instance of the opal product.

### CODE REVIEW

**Review by MJ-USAF — DISMISSED** [review]
I think this accurately reflects the steps we took to locally deploy opal earlier today. Thanks Mike!

File: README.md [hunk 1]
```diff
@@ -8,23 +8,42 @@ This repo contains all the deployment code and configuration files necessary for
...
+    **Base DNS name of this deployment (i.e. .companyname.com)**
+        .127.0.0.1.nip.io 
+    **Add opal to base URL? (y,n)**
+        enter n
+    **Banner color (Looks like classification banner, must be HTML color):**
+        this can be any color to the user's preference green, blue, red, etc.
+    **Banner name (Can be used for network name or classification markings)**
+        this can be any text according the the preference of the user.
+    **Deploy keycloak with OPAL (y/n)**
+        Always choice y, unless you are an expert with OPAL and have an external keycloak deployment.
```
Comment (AB-USAF) [inline README.md:35]: This is hard to read when not in code view

**Review by AB-USAF — APPROVED** [review]
It looks good, follows all the steps the team followed yesterday. But, point 8 was hard to read when viewing the README.md preview.

**Review by AB-USAF — APPROVED** [review]
:goat:

**Review by MJ-USAF — APPROVED** [review]

**Review by JA-USAF — APPROVED** [review]

**Pull request merged**